tutorials/inputs/controllers_gamepads_joysticks.rst

```diff
@@ -0,0 +1,246 @@
+.. _doc_controllers_gamepads_joysticks:
+
+Controllers, gamepads, and joysticks
+====================================
+
+Godot supports hundreds of controller models thanks to the community-sourced
+`SDL game controller database <https://github.com/gabomdq/SDL_GameControllerDB>`__.
+
+Controllers are supported on Windows, macOS, Linux, Android, iOS, and HTML5.
+
+Note that more specialized devices such as steering wheels, rudder pedals and
+`HOTAS <https://en.wikipedia.org/wiki/HOTAS>`__ are less tested and may not
+always work as expected. If you have access to one of those devices, don't hesitate to
+`report bugs on GitHub <https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#reporting-bugs>`__.
+
+In this guide, you will learn:
+
+- **How to write your input logic to support both keyboard and controller inputs.**
+- **How controllers can behave differently from keyboard/mouse input.**
+- **Troubleshooting issues with controllers in Godot.**
+
+Supporting universal input
+--------------------------
+
+Thanks to Godot's input action system, Godot makes it possible to support both
+keyboard and controller input without having to write separate code paths.
+Instead of hardcoding keys or controller buttons in your scripts, you should
+create *input actions* in the Project Settings which will then refer to
+specified key and controller inputs.
+
+Input actions are explained in detail on the :ref:`doc_inputevent` page.
+
+.. note::
+
+    Unlike keyboard input, supporting both mouse and controller input for an
+    action (such as looking around in a first-person game) will require
+    different code paths since these have to be handled separately.
+
+Which Input singleton method should I use?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In Godot 4.0, there are 3 ways to get input in an analog-aware way:
+
+- When you have two axes (such as joystick or WASD movement) and want both
+  axes to behave as a single input, use ``Input.get_vector()``:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    # `velocity` will be a Vector2 between `Vector2(-1.0, -1.0)` and `Vector2(1.0, 1.0)`.
+    var velocity = Input.get_vector("move_left", "move_right", "move_forward", "move_back")
+
+    # The line above is a shorter form of:
+    var velocity = Vector2(Input.get_action_strength("move_right") - Input.get_action_strength("move_left"),
```

Comment:
This is false in fact. One of the advantages of the using get_vector over this is the handling of the deadzone.
With this solution the deazone end up square instead of circular.

I'm ok with mentionning that they are similar bu it's false to say they are equivalent.

Reply:
PR opened: https://github.com/godotengine/godot-docs/pull/4620#discussion_r575847917